packages/ui/src/views/repo/pull-request/details/components/conversation/pull-request-comment-box.tsx

```diff
@@ -174,43 +213,33 @@ const PullRequestCommentBox = ({
             </div>
           </TabsContent>
         </Tabs>
-        <div className="mt-4 flex items-center gap-x-3">
-          {!inReplyMode && !isEditMode ? (
-            <Button onClick={handleSaveComment}>Comment</Button>
-          ) : isEditMode ? (
-            <>
-              <Button onClick={handleSaveComment}>Save</Button>
-              <Button variant="outline" onClick={onCancelClick}>
-                Cancel
-              </Button>
-            </>
-          ) : (
-            <></>
-          )}
-
-          {inReplyMode && (
-            <>
-              {isEditMode ? (
-                <Button onClick={handleSaveComment}>Save</Button>
-              ) : (
-                <Button onClick={handleSaveComment}>Reply</Button>
-              )}
-              <Button
-                variant={'outline'}
-                onClick={() => {
-                  if (comment.trim()) {
-                    onCommentSaveAndStatusChange?.(comment.trim(), isResolved ? 'active' : 'resolved', parentCommentId)
-                    onCancelClick?.()
-                  }
-                }}
-              >
-                {isResolved ? 'Reply & Reactivate' : 'Reply & Resolve'}
-              </Button>
+
+        <div className="mt-4 flex items-center justify-between">
+          <div>
+            <input type="file" ref={fileInputRef} className="hidden" onChange={handleFileChange} />
+            <Button
+              className="gap-x-2 px-2.5 font-normal text-foreground-3 hover:bg-background-8"
+              variant="custom"
+              onClick={handleFileSelect}
+            >
+              <Icon size={16} name="attachment-image" />
```

Comment:
drag and drop shud be hidden when preview tab is selected also shud hide comment in our case 
![image](https://github.com/user-attachments/assets/1da977fb-c3fa-412d-b281-36fb1525a6e6)
@andrewgolovanov